BAKKA_Editor/Assets/locales/en-US.axaml

```diff
@@ -2,6 +2,8 @@
                     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
                     xmlns:system="clr-namespace:System;assembly=System.Runtime">
     <!-- TODO: namespaces, ordering, and grouping -->
+    <system:String x:Key="L.ApplicationVersion">BAKKA Avalonia - v2.19.0 Nightly 1</system:String>
```

Comment:
no bueno, definitely not here

Reply:
putting it in the locales is definitely dumb, should probably just have an application version string in app.axaml somewhere else.